hack/update-manifests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Interesting. Why the change to bash? The script looks pretty much POSIXish.

Reply:
Actually it probably doesn't need to change. I think I was experimenting and never changed it back. Will update.